tools/remark-reading-time.mjs

```diff
@@ -0,0 +1,89 @@
+import getReadingTime from "reading-time";
+import { toString } from "mdast-util-to-string";
+import { filter } from "unist-util-filter";
+
+import assert from "node:assert/strict";
+
+const WORDS_PER_MINUTE = 265;
+
+const getSecondsAddedByImages = (imageCount) => {
+  const nbOfImagesBellow10 = Math.max(Math.min(imageCount, 10), 0);
+  const secondsAddedByImagesBellow10 =
+    12 * nbOfImagesBellow10 -
+    (nbOfImagesBellow10 * (nbOfImagesBellow10 - 1)) / 2;
+
+  const nbOfImagesAbove10 = Math.max(imageCount - 10, 0);
+  const secondsAddedByImagesAbove10 = nbOfImagesAbove10 * 3;
+
+  return secondsAddedByImagesBellow10 + secondsAddedByImagesAbove10;
+};
+let totalForTest = 0;
+assert.equal(getSecondsAddedByImages(-1), (totalForTest += 0));
+assert.equal(getSecondsAddedByImages(0), (totalForTest += 0));
+assert.equal(getSecondsAddedByImages(1), (totalForTest += 12));
+assert.equal(getSecondsAddedByImages(2), (totalForTest += 11));
+assert.equal(getSecondsAddedByImages(3), (totalForTest += 10));
+assert.equal(getSecondsAddedByImages(4), (totalForTest += 9));
+assert.equal(getSecondsAddedByImages(5), (totalForTest += 8));
+assert.equal(getSecondsAddedByImages(6), (totalForTest += 7));
+assert.equal(getSecondsAddedByImages(7), (totalForTest += 6));
+assert.equal(getSecondsAddedByImages(8), (totalForTest += 5));
+assert.equal(getSecondsAddedByImages(9), (totalForTest += 4));
+assert.equal(getSecondsAddedByImages(10), (totalForTest += 3));
+assert.equal(getSecondsAddedByImages(11), (totalForTest += 3));
```

Comment:
If it is wrong, it'll crash at compile time:

<img width="294" alt="image" src="https://github.com/user-attachments/assets/49ca8e99-a8c9-46de-be35-620a46b1a765" />